### PR TITLE
[#11152] Fix bug that caused SVGs to fail to render

### PR DESCRIPTION
**Pull request by jianhandev**
Part of #11152 

Fixes the bug that caused SVGs to not be rendered even in browsers which has support for `image/svg+xml`.

https://user-images.githubusercontent.com/59337195/120337974-5eb49a80-c326-11eb-8926-345f24ae3c19.mov

### CODE REVIEW

File: src/main/java/teammates/ui/webapi/ImageResult.java [hunk 1]
```diff
@@ -29,7 +32,11 @@
 
     @Override
     void send(HttpServletResponse resp) throws IOException {
-        resp.setContentType("image/png");
+        String contentType = guessContentTypeFromStream(new ByteArrayInputStream(bytes));
+        if ("application/xml".equals(contentType)) {
+            contentType = "image/svg+xml";
```
[the remark this answers is not in the document]
Comment (jianhandev) [reply src/main/java/teammates/ui/webapi/ImageResult.java:37]: This is a hacky method as `guessContentTypeFromStream` does not guess specifically "image/svg+xml", but only image files are saved to storage in `PostStudentProfilePictureAction.java`, and the only image mime type with xml is svg.
https://github.com/TEAMMATES/teammates/blob/99f62a55e3371ba0d7376c8bcb6b54aa342cc2e1/src/main/java/teammates/ui/webapi/PostStudentProfilePictureAction.java#L44-L47

File: src/main/java/teammates/ui/webapi/ImageResult.java [hunk 2]
```diff
@@ -1,5 +1,8 @@
 package teammates.ui.webapi;
 
+import static java.net.URLConnection.guessContentTypeFromStream;
```
[the remark this answers is not in the document]
Comment (jianhandev) [reply src/main/java/teammates/ui/webapi/ImageResult.java:3]: You're right static import can be avoided here.

File: src/main/java/teammates/ui/webapi/ImageResult.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (wkurniawan07) [reply src/main/java/teammates/ui/webapi/ImageResult.java:36]: That assumes the above is the only entrypoint that writes files to the image storage. It is however a reasonable assumption.
Then you should add a comment to explain why this mime type conversion needs to be done.

**Review by wkurniawan07 — APPROVED** [review]
LGTM

**Pull request merged**